adcm_client/objects.py

```diff
@@ -420,16 +420,16 @@ def host_delete(self, host: "Host"):
             self._subcall("host", "delete", host_id=host.id)
 
     def service(self, **args) -> "Service":
-        return self._subobject(Service, **args)
+        return self._child_obj(Service, **args)
 
     def service_list(self, paging=None, **args) -> "ServiceList":
-        return self._subobject(ServiceList, paging=paging, **args)
+        return self._child_obj(ServiceList, paging=paging, **args)
 
     def service_add(self, **args) -> "Service":
         proto = self.bundle().service_prototype(**args)
```

Comment:
Please move this line under allure_step. Dunno why it is not under the context manager.

Reply:
Then you will have to remove from the description `with allure_step("Add service {} to cluster {}".format(proto.name, self.name)):` `proto.name`